tests/rpm.bats

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "rpm-build" {
+	if ! which runc ; then
+		skip
+	fi
+
+	# Build the container.
+	image=fedora:26
+	cid=$(buildah --debug=false from --pull --signature-policy ${TESTSDIR}/policy.json $image)
+	root=$(buildah --debug=false mount $cid)
+	commit=$(git log --format=%H -n 1)
+	shortcommit=$(echo ${commit} | cut -c-7)
+	mkdir -p ${root}/rpmbuild/{SOURCES,SPECS}
+
+	# Build the tarball.
+	(cd ..; git archive --format tar.gz --prefix=buildah-${commit}/ ${commit}) > ${root}/rpmbuild/SOURCES/buildah-${shortcommit}.tar.gz
+
+	# Update the .spec file with the commit ID.
+	sed s:REPLACEWITHCOMMITID:${commit}:g ${TESTSDIR}/../contrib/rpm/buildah.spec > ${root}/rpmbuild/SPECS/buildah.spec
+
+	# Install build dependencies and build binary packages.
+	buildah --debug=false run $cid -- dnf -y install 'dnf-command(builddep)' rpm-build
+	buildah --debug=false run $cid -- dnf -y builddep --spec rpmbuild/SPECS/buildah.spec
+	buildah --debug=false run $cid -- rpmbuild --define "_topdir /rpmbuild" -ba /rpmbuild/SPECS/buildah.spec
+
+	# Save copies of the binary packages.
+	rpmdir=$(mktemp -d ${TESTDIR}/rpmsXXXXXXXX)
+	cp ${root}/rpmbuild/RPMS/*/*.rpm ${rpmdir}/
+
+	# Build a new container.
+	buildah rm $cid
+	cid=$(buildah --debug=false from --pull --signature-policy ${TESTSDIR}/policy.json fedora:26)
```

Comment:
Something to consider.  Perhaps change the name of this variable to cid2 to make it abundantly clear?  Your call.